tooling/rib-intellij-plugin/build.gradle

```diff
@@ -197,19 +197,22 @@ afterEvaluate {
     }
 }
 
-uploadArchives {
+publishing {
     repositories {
-        mavenDeployer {
-            repository url: "file://" + new File(System.getProperty("user.home"), ".m2/repository").absolutePath
+        maven {
             if (isReleaseBuild() && hasMavenCredentials()) {
-                repository(url: "${config.build.artifactoryUrl}/${pluginsMavenRepositoryBucket}") {
-                    authentication(userName: mavenUser, password: mavenPassword)
+                url = "${config.build.artifactoryUrl}/${pluginsMavenRepositoryBucket}"
+                credentials {
+                    username = mavenUser
+                    password = mavenPassword
                 }
+            } else {
+                url = "file://" + new File(System.getProperty("user.home"), ".m2/repository").absolutePath
```

Comment:
This branch is unnecessary as you can always publish there using `publishToMavenLocal`. I suggest rewriting like this:
```
if (hasMavenCredentials()) {
    maven {
        ...
    }
}
```